components/rmrk/Gallery/Search/SearchBar.vue

```diff
@@ -133,7 +132,7 @@ export default class SearchBar extends mixins(PrefixMixin) {
     search: '',
     type: '',
     sortBy: 'BLOCK_NUMBER_DESC',
-    listed: false,
+    listed: true,
```

Comment:
same as above

Reply:
here i can leave it on `false`. but on gallery it should be true by default